pre-commit.yml

```diff
@@ -0,0 +1,14 @@
+name: pre-commit
```

Comment:
Isn't that required to go into `.github/workflows` ?

Reply:
Yes 🤦🏼